src/bicep/modules/SentinelSolution.bicep

```diff
@@ -0,0 +1,20 @@
+param workspaceName  string
+param workspaceId string
+param location string = resourceGroup().location
```

Comment:
One thing I did run into was that my LAWS resource was in Central US, but the resourceGroup().location was in East US so the deployment would always fail with 'Not Found'.

Let's take in another parameter `workspaceLocation` that ensures we create the Sentinel solution in the same region as the LAWS resource.